libs/remote_storage/src/azure_blob.rs

```diff
@@ -312,3 +318,143 @@ impl RemoteStorage for AzureBlobStorage {
         Ok(())
     }
 }
+
+pin_project_lite::pin_project! {
```

Comment:
offtopic: Are there any benefits for us using `pin_project_lite` rather than `pin_project` ? (IIRC, the listed benefits are typically if you don't already heavily use `syn`)

Reply:
I don't know, I just remembered that we only had `lite` here in `remote_storage` and not pin_project..